### PR TITLE
fix readUnicodeStringBytes when decode chinese

### PR DESCRIPTION
**Pull request by Yong-ee**

### CODE REVIEW

File: nbfx/decoder.go [hunk 1]
```diff
@@ -244,7 +244,7 @@ func readUnicodeStringBytes(r io.Reader, len uint32) (string, error) {
 		if err != nil {
 			return string(runes), err
 		}
-		var runeInt int16
+		var runeInt uint16
```
Comment (khoad) [inline nbfx/decoder.go:247]: Can you write a unit test for this change?

**Comment by timerickson** [comment]
Thanks for the contrib, @Yong-ee ! Can you add a test for coverage? Also, why did the Travis CI build fail, @khoad ? I don't see why - just that it failed. The details don't help.

**Comment by Yong-ee** [comment]
hi, @timerickson i add a test **TestDecodeExampleUnicodeChars16TextWithChinese**, the _runeInt_ will be overflow when decode chinese use **readUnicodeChars16Text** func，the test is an example.The CI failed because `flag needs an argument: -repotoken`

**Review by khoad — APPROVED** [review]

**Comment by khoad** [comment]
I've fixed the build issue and approved the PR.
